jadx-gui/src/main/java/jadx/gui/JadxGUI.java

```diff
@@ -42,6 +43,8 @@ public static void main(String[] args) {
 				mw.init();
 				registerOpenFileHandler(mw);
 			});
+
+			JadxPluginsTools.getInstance().updateAll();
```

Comment:
This action should be configurable in preferences, executed in background task and reload window on finish if any plugin was updated.